tests/comfy_cli/cql/test_gallery.py

```
@@ -0,0 +1,152 @@
+    assert gallery.filter_rows(rows, type_="video") == [r for r in rows if r["name"] == "video_kling"]
+    assert gallery.filter_rows(rows) == rows  # no predicates → identity
+
+
+def test_load_gallery_from_explicit_path(tmp_path):
+    p = tmp_path / "index.json"
+    p.write_text(json.dumps(CATEGORIES[:1]))
+    cats = gallery.load_gallery(str(p))
+    assert cats[0]["title"] == "Image"
+
+
+def test_load_gallery_fetches_and_caches(tmp_path, monkeypatch):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)